src/redeem/redeemDataService.ts

```diff
@@ -125,7 +125,7 @@ export async function getRecentDailyRedeems(
             `,
                 [daysBack]
             )
-        ).rows.map((row) => ({ date: row.date, sat: row.sat }));
+        ).rows.map((row) => ({ ...row, sat: Number(row.sat) }));
```

Comment:
Satoshis might not fit in `number`. Better to use `BN`

Reply:
I checked and `100_000_000 * 21_000_000 < Number.MAX_SAFE_INTEGER` so it should be good.